addons/account/models/account_move.py

```diff
@@ -1161,6 +1162,11 @@ def onchange(self, values, field_name, field_onchange):
     # COMPUTE METHODS
     # -------------------------------------------------------------------------
 
+    @api.depends('move_type', 'company_id')
+    def _compute_is_storno(self):
+        for move in self:
+            move.is_storno = move.is_storno or (move.move_type in ('out_refund', 'in_refund') and move.company_id.storno_accounting)
```

Comment:
This doesn't work for boolean fields
Once the field is true, it will be impossible to set it to false because of the `move.is_storno or`

One solution can be to use a Selection field

Reply:
Although this is perfectly right, I don't see how this could happen: once the field has been set, it won't change because the move_type can't be changed, nor the company. :thinking: 

and the Storno setting is supposed to be set at install time.

so it could stay as it is, probably

Reply:
but @api.depends('move_type', 'company_id.storno_accounting') would be more accurate

Reply:
`company_id.storno_accounting` has been removed from the `@api.depends`
The user should be able to switch from Storno to non-Storno and vice-versa.
If we keep this dependency, the previous `account move line`'s will be updated even though they should not (it would break the `check_credit_debit` verification because a previous non-Storno move will become Storno but the debit/credit values would not be updated therefore resulting in an error)